kcidb/test_oo.py

```diff
@@ -0,0 +1,454 @@
+"""kcidb.oo test module"""
+
+import kcidb
+from kcidb import db
+from kcidb import oo
+
+
+class KCIDB00ClientTestCase(kcidb.unittest.TestCase):
+    """Test case for kcidb.oo module"""
+
+    def setUp(self):
+        """Database setup and oo client creation"""
+        self.source = db.Client('sqlite::memory:')
+        self.source.init()
+
+        self.initialize_and_load_data()
+        self.assertTrue(isinstance(self.source, kcidb.orm.Source))
```

Comment:
If we need that test, then it shouldn't be here, but in the "orm" test instead. Please remove.